parm/config/config.base.emc.dyn

```diff
@@ -354,13 +354,13 @@ export MAKE_NSSTBUFR="@MAKE_NSSTBUFR@"
 export MAKE_ACFTBUFR="@MAKE_ACFTBUFR@"
 
 # Analysis increments to zero in CALCINCEXEC
-export INCREMENTS_TO_ZERO="'liq_wat_inc','icmr_inc'"
+export INCREMENTS_TO_ZERO="'liq_wat_inc','icmr_inc','rwmr_inc','snmr_inc','grle_inc'"
```

Comment:
If this is a production update, shouldn't these changes also be made to `config.base.nco.static`?

Reply:
Yes, good catch. I have now also updated `config.base.nco.static` for the INCREMENTS variables @ 79d08613. Even though `config.base.nco.static` needs a full review and possible redo it's worth keeping it up-to-date with these changes. Thanks @WalterKolczynski-NOAA !